dashboard/app/views/gallery_activities/index.html.haml

```diff
@@ -9,8 +9,13 @@
     = link_to t('gallery.students_made_these_with_code'), gallery_activities_path
 
 - if params[:app] # pagination page: just one type of activity
-  -# TODO: bring back pagination and make it efficient
   = render partial: 'gallery_activities', locals: {gallery_activities: @gallery_activities}
+  .span{style: 'float:right;'}
+    - page = params[:page] || 1
+    - page = page.to_i
+    - if page < 10
```

Comment:
should 10 be a constant somewhere? and should we also 404 requests with pages > 10?